ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-release-4.6.yaml

```diff
@@ -69,7 +69,7 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts make test-e2e-olm
+      commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts make test-e2e-olm E2E_TEST_INCLUDES=logforwarding CLF_TEST_INCLUDES=elasticsearchmanaged
```

Comment:
why would the ES proxy need the log forwarding tests?

Reply:
corrected.  it was on the wrong job

Reply:
i'm still seeing it on the proxy jobs, they shouldn't need to do the log-forwarding tests. it also looks like the elasticsearch operator job configs have this as well... to me this defeats the purpose of what we're doing because it is configured to work exactly the same